src/OpenLoco/src/Logging.cpp

```diff
@@ -28,18 +29,24 @@ namespace OpenLoco::Diagnostics::Logging
     static void cleanupLogFiles(const fs::path& logsFolderPath)
     {
         // Get all log files in the folder
-        std::vector<std::filesystem::path> logFiles;
-        for (const auto& file : std::filesystem::directory_iterator(logsFolderPath))
+        std::vector<fs::path> logFiles;
+        try
         {
-            if (file.path().extension() == ".log")
+            for (const auto& file : fs::directory_iterator(logsFolderPath))
             {
-                logFiles.push_back(file.path());
+                if (file.path().extension() == ".log")
+                {
+                    logFiles.push_back(file.path());
+                }
             }
         }
+        catch (...)
```

Comment:
can we just catch the `std::filesystem::filesystem_error` and state that its probably due to the folder not existing?

Reply:
Not sure what that would achieve, its just supposed to give a list of files and if its not accessible by any means it should be simply empty, this in my opinion better expresses that we don't care about the exceptions it would potentially throw.